zrml/prediction-markets/src/lib.rs

```diff
@@ -2035,7 +2038,11 @@ mod pallet {
         ) -> DispatchResult {
             ensure!(market.report.is_some(), Error::<T>::MarketNotReported);
             Self::ensure_outcome_matches_market_type(market, outcome)?;
-            Self::ensure_can_not_dispute_the_same_outcome(disputes, outcome)?;
+            Self::ensure_can_not_dispute_the_same_outcome(
+                disputes,
+                &market.report.as_ref().unwrap(),
+                outcome
+            )?;
             Self::ensure_disputes_does_not_exceed_max_disputes(num_disputes)?;
             Ok(())
```

Comment:
Please increment `spec_version` in `runtime/src/lib.rs` by one to signal changes in the runtime logic.